fudy/NoteScreen/Component/FudyButton.qml

```diff
@@ -1,31 +1,51 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.12
```

Comment:
remove version

Reply:
I will fix it in a commit for refactoring.